### PR TITLE
refactor learn more TV to reuse card_reader_learn_more_section.xml

### PR DESCRIPTION
**Pull request by rossanafmenezes**
Closes: # N.A

### Description

Refactoring the Material Text view used [here](https://github.com/woocommerce/woocommerce-android/pull/8307) to use card_reader_learn_more_section.xml and fix the animation and alignment issues.

### Testing instructions

1. Go to Menus
2. Go to Payments
3. Check if the learn more view shows underneath the rows
4. test on landscape mode, make sure it does not disappear
5. enable and disable Pay in Person option, make sure the Learn More view is displayed in both states
6. Click on the Learn More option
7. make sure it takes you to the right webview
9. Back to the payments menu create a simple payment
10. select collect payment
11. Make sure the learn more link also shows there

### Images/gif

Screen Shot 2023-02-23 at 4 55 20 PM

Screen Shot 2023-02-23 at 4 55 47 PM

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/SelectPaymentMethodFragment.kt [hunk 1]
```diff
@@ -153,10 +153,9 @@ class SelectPaymentMethodFragment : BaseFragment(R.layout.fragment_take_payment)
         }
 
         with(binding.learnMoreIppPaymentMethodsTv) {
-            val learMoreIpp = state.learMoreIpp
-            setOnClickListener { learMoreIpp.onClick.invoke() }
-            UiHelpers.setTextOrHide(this, learMoreIpp.label)
+            learnMore.setOnClickListener { state.learMoreIpp.onClick.invoke() }
         }
+        UiHelpers.setTextOrHide(binding.learnMoreIppPaymentMethodsTv.learnMore, state.learMoreIpp.label)
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/SelectPaymentMethodFragment.kt]: np: Why did you move this out of the `with` block? `With` block with just one line inside feels redundant. This also probably will work:

```
        with(binding.learnMoreIppPaymentMethodsTv) {
            learnMore.setOnClickListener { state.learMoreIpp.onClick.invoke() }
            UiHelpers.setTextOrHide(learnMore, state.learMoreIpp.label)
        }
```

**Comment by kidinov** [comment]
Not sure if that's something we need to address. I do not entirely understand why this behaves this way 🤔. I'll leave it up to you if you want to do anything with this =)



https://user-images.githubusercontent.com/4923871/221104918-dadeea5b-f407-4776-9c75-3ba1f7fc9d02.mp4

**Review by kidinov — APPROVED** [review]
LGTM! I left a couple of not blocking comments

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/SelectPaymentMethodFragment.kt (hunk 1 above)
Comment (rossanafmenezes) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/SelectPaymentMethodFragment.kt]: fixed here [921772a](https://github.com/woocommerce/woocommerce-android/pull/8393/commits/921772af6cde6ce4abff4bfef2b13c23232c05a2)

**Comment by rossanafmenezes** [comment]
> Not sure if that's something we need to address. I do not entirely understand why this behaves this way 🤔. I'll leave it up to you if you want to do anything with this =)

Thanks, @kidinov 

Isn't it because there's a recycler view inside the scroll view? So when on a smaller screen the RV rolls first then the scroll view rolls? I'm not sure. 

The issue is that removing the scrollview won't work on small screens or LS mode. Not only for the learn more link. The failed onboarding banner won't show either on LS mode (I realized that when I was working on this PR p1675436126786089/1675344960.465179-slack-C025A8VV728 ). So I guess we could redo the whole learn more row as part of the recycler view but it won't fix the issue for the banner in any case. 

Any ideas?

**Comment by kidinov** [comment]
@rossanafmenezes 

> Isn't it because there's a recycler view inside the scroll view? So when on a smaller screen the RV rolls first then the scroll view rolls? I'm not sure.

Yep, it probably is

As you said we could put learn more inside RV, and remove ScrollView. We can put an onboarding banner bellow RV so it will be always visible when needed. All of this can be done as part of another PR though

**Comment by rossanafmenezes** [comment]
> @rossanafmenezes
> 
> > Isn't it because there's a recycler view inside the scroll view? So when on a smaller screen the RV rolls first then the scroll view rolls? I'm not sure.
> 
> Yep, it probably is
> 
> As you said we could put learn more inside RV, and remove ScrollView. We can put an onboarding banner bellow RV so it will be always visible when needed. All of this can be done as part of another PR though

Ok. I will merge this one, then. Thanks
